src/Extensions/AzureBlobStorage/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -9,36 +9,28 @@
 using Azure.Storage.Blobs.Models;
 using Azure.Storage.Blobs.Specialized;
 using Azure.Storage.Queues;
-using Microsoft.Extensions.Logging;
-using System;
-using System.Collections.Generic;
 using System.Globalization;
-using System.IO;
 using System.Net;
-using System.Threading;
-using System.Threading.Tasks;
 
-namespace Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob
+namespace Microsoft.Diagnostics.Monitoring.AzureStorage.AzureBlob
 {
     /// <summary>
     /// Egress provider for egressing stream data to an Azure blob storage account.
     /// </summary>
     /// <remarks>
     /// Blobs created through this provider will overwrite existing blobs if they have the same blob name.
     /// </remarks>
-    internal partial class AzureBlobEgressProvider :
-        EgressProvider<AzureBlobEgressProviderOptions>
+    internal partial class AzureBlobEgressProvider
     {
-        private int BlobStorageBufferSize = 4 * 1024 * 1024;
+        private readonly string AzureBlobStorage = "AzureBlobStorage";
```

Comment:
This should be moved to some kind of shared constant because it is used in the same way both here and in the Program class.